# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/gfx-backend-dx12/.cargo-checksum.json
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/gfx-backend-dx12/.cargo-checksum.json b/third_party/rust/gfx-backend-dx12/.cargo-checksum.json
deleted file mode 100644
--- a/third_party/rust/gfx-backend-dx12/.cargo-checksum.json
+++ /dev/null
@@ -1,1 +0,0 @@
-{"files":{"Cargo.toml":"a735bb8cd1bb9ac621a7db1964931bf2c7b17f111f34d98381a4e5ff8c527f81","README.md":"a92efd814ab7708898fd26c7378ac3bacbfa4e670c670d316aaa6c1402a0046c","shaders/blit.hlsl":"1f8819f3a91acf71a69bfd14ccd245180a92a9efb0ab76bf6b2e89aae10f3952","src/command.rs":"7b79e68e4086341ac53170af8081045762109bf8a5447056a895bdc5000a6c1a","src/conv.rs":"b367d52fb4775afd34e266a81a9d0af4a3707b95a6ddf5fa4a9db6cf0e1b1eda","src/descriptors_cpu.rs":"36bb020dac945f817813beecbfb44bda9d0c68ca51a1aa8aa35f661c083692d9","src/device.rs":"c16c6cd1790882e1af1e2ed6fb9abeb25a37b28cd0f21d5d773061282bde48c1","src/internal.rs":"c91b9fcbaa681bda3894f13d459e8c4152afd84accac75b98bf635767e85fe50","src/lib.rs":"2e3171989ffba5fe511000aea5b0334b4b57dfc7ce03385997396256454074e3","src/pool.rs":"f3c55543190e18353eb37088fc6169ea3d9af93cfc79f726c1a619d4649beaa4","src/resource.rs":"3b9cbe723cc72e4e10456d9715969f6a0cbb1b50cfb446701c97f4c517e2eac0","src/root_constants.rs":"9bf4b8cc24ee7ceec0de5f3c293a995ee8e9cf4526ef2b80c95514bf6f76cb2f","src/window.rs":"55015d626f4a2094b10753c169c1bbdece5e8e30869d0f18dae8286168ea4d5c"},"package":"21506399f64a3c4d389182a89a30073856ae33eb712315456b4fd8f39ee7682a"}
\ No newline at end of file